mgmt_cli_test.py

```diff
@@ -182,11 +182,11 @@ def restore_backup_from_backup_task(self, mgr_cluster, backup_task, keyspace_and
     # pylint: disable=too-many-arguments
     def verify_backup_success(self, mgr_cluster, backup_task, ks_names: list = None, tables_names: list = None,
                               truncate=True, restore_data_with_task=False, timeout=None):
-        if ks_names is None:
```

Comment:
```
Fixed list mutation issue in verify_backup_success method. 
The previous implementation changed the initial lists ks_names and tables_names...
```
I don't see mutation of the original object here.
I see a change of behavior for empty input list.
When the input value is `None` then we create new `list` object and don't mutate the immutable `None`.

```
. Since the same list was used in the caller method, it resulted in unintended behavior.
```

The diff is in the behavior in this method, why the commit/PR refers to a caller method?
If the point of this commit/PR is in the change of the empty list behavior then exactly this should be stated.
For now it is confusing.